serverless-logic/serverless-logic-managing-services/serverless-logic-configuring-openAPI-services-endpoints.adoc

```diff
@@ -0,0 +1,101 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="serverless-logic-configuring-openAPI-services-endpoint"]
+= Configuring OpenAPI services endpoints
+:context: serverless-logic-configuring-openAPI-services-endpoints
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+{ServerlessLogicProductName} uses the `kogito.sw.operationIdStrategy` property to generate the REST client for invoking services defined in OpenAPI documents. This property determines how the configuration key is derived for the REST client configuration.
+
+The `kogito.sw.operationIdStrategy` property supports the following values:
+
+* `FILE_NAME` This is the default value.
+* `FULL_URI`
+* `FUNCTION_NAME`
+* `SPEC_TITLE`
```

Comment:
I'm confused why there's a bulleted list, and then expanded detail about all four items directly below this. 

Maybe try the following: 

```suggestion
The `kogito.sw.operationIdStrategy` property supports the following values: `FILE_NAME`, `FULL_URI`, `FUNCTION_NAME`, and `SPEC_TITLE`.
```